### PR TITLE
[EDU-3247] feat: origin CRUD cli

### PR DESCRIPTION
**Pull request by gabriel-azion**
https://aziontech.atlassian.net/browse/EDU-3247

### CODE REVIEW

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 1]
```diff
@@ -0,0 +1,246 @@
+---
+title: Origins
+description: Create, list, describe, update, and delete your origins on the Azion platform through Azion CLI.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:3]:
```suggestion
description: Create, list, describe, update, and delete your origins on Azion's platform through Azion CLI.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+
+The `--origin-path` flag defines the path to be appended to the URI when forwarding the request to the origin. Leave it blank to use only the URI.
+
+#### origin-protocol-policy
+
+The `--origin-protocol-policy` flag tells the protocol policy used in the origin.
+
+#### origin-type
+
+The `--origin-type` flag identifies the source of a record, i.e., `single_origin`.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:68]:
```suggestion
The `--origin-type` flag identifies the source of a record. Example: `single_origin`.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+```bash
+$ azion list origin --application-id <application-id>
+```
+
+### Required flags
+
+#### application-id
+
+The `--application-id` flag sets the unique identifier for the edge application. It is a required field.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:88]:
```suggestion
The `--application-id` flag sets the unique identifier for the edge application. It's a required field.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+
+The `--filter` flag filters items by their name.
+
+#### order-by
+
+The `--order-by` flag sorts the output based on the selected field.
+
+#### page
+
+The `--page` flag returns a specific page of the list according to its number. The default is 1.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:106]:
```suggestion
The `--page` flag returns a specific page of the list according to its number. The default is `1`.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+
+#### page-size
+
+The `--page-size` flag defines how many items should be returned per page. The default is 10.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:110]:
```suggestion
The `--page-size` flag defines how many items should be returned per page. The default is `10`.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+
+#### sort
+
+The `--sort` flag defines the order of the items on the list; options are `<asc|desc>`.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:114]:
```suggestion
The `--sort` flag defines the order of the items on the list. The value should be either `asc` or `desc`.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:114]: following the example in cache settings comm ref doc.

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+
+```bash
+$ azion describe origin --application-id <application-id> --origin-key <origin-key> 
+```
+
+### Required flags
+
+#### application-id
+
+The `--application-id` flag sets the unique identifier for an edge application. It is a mandatory field.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:134]:
```suggestion
The `--application-id` flag sets the unique identifier for an edge application. It's a mandatory field.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 8]
```diff
@@ -0,0 +1,246 @@
...
+
+#### origin-key
+
+The `--origin-key` flag sets the unique identifier for an origin. It is a mandatory field.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:138]:
```suggestion
The `--origin-key` flag sets the unique identifier for an origin. It's a mandatory field.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 9]
```diff
@@ -0,0 +1,246 @@
...
+### Required flags
+
+#### application-id
+
+The `--application-id` flag sets the unique identifier for an edge application. It is a required field.
+
+#### origin-key
+
+The `--origin-key` flag sets the unique identifier for an origin. It is a required field.
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:173]:
```suggestion
The `--application-id` flag sets the unique identifier for an edge application. It's a required field.

#### origin-key

The `--origin-key` flag sets the unique identifier for an origin. It's a required field.

```

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 10]
```diff
@@ -0,0 +1,246 @@
...
+The `--addresses` flag passes a list of addresses linked to the origin. Multiple addresses should be separated by commas.
+
+#### host-header
+
+The `--host-header` flag specifies the hostname of the server being accessed.
+
+#### origin-type
+
+The `--origin-type` flag identifies the source of a record, i.e., `single_origin`.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:190]:
```suggestion
The `--origin-type` flag identifies the source of a record. Example: `single_origin`.
```

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 11]
```diff
@@ -0,0 +1,246 @@
...
+
+### Required flags
+
+#### application-id
+
+The `--application-id` flag sets the unique identifier for an edge application. It is a required field.
+
+#### origin-key
+
+The `--origin-key` flag sets the unique identifier for an origin. It is a required field.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:242]:
```suggestion
The `--application-id` flag sets the unique identifier for an edge application. It's a required field.

#### origin-key

The `--origin-key` flag sets the unique identifier for an origin. It's a required field.
```

**Review by MarianaAguilera — APPROVED** [review]
a few style suggestions.

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx (hunk 2 above)
Comment (hannah-sk) [reply src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:68]: isso, não usamos i.e.

File: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx [hunk 12]
```diff
@@ -0,0 +1,246 @@
...
+
+```bash
+$ azion update origin --application-id <application-id> --origin-key <origin-key> 
+```
+
+### Required flags
+
+#### application-id
+
+The `--application-id` flag sets the unique identifier for an edge application. It is a required field.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/devtools/cli/azion-cli-alpha/origins/origins.mdx:168]:
```suggestion
The `--application-id` flag sets the unique identifier for an edge application. It 's a required field.
```

**Review by hannah-sk — APPROVED** [review]

**Pull request merged**